src/main/java/com/cmsc495/spacesim/model/Persons.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package com.cmsc495.spacesim.model;
+
+/**
+ *
+ * @author NoorQasim
+ */
+public class Persons {
+    
+    private String[] skill = {"Food", "Water", "Gasoline", "Coal", "Medical Supplies", "Builidng Materials"};

```

Comment:
This skill field on this class should just be a string instead of an array, and the array should instead be a public+static array with a name like `allPossibleSkills`. As it is now, every new Person object would have all the skills instead of each having a single skill which we determine when we construct the object.